src/app/tests/suites/certification/Test_TC_CGEN_2_1.yaml

```diff
@@ -66,12 +67,30 @@ tests:
               maxValue: 2
 
     #Issue 17997
-    - label: "TH1 reads BasicCommissioningInfo attribute from DUT"
+    - label:
+          "TH1 reads BasicCommissioningInfo attribute from DUT and Verify that
+          the BasicCommissioningInfo attribute has the following field:
+          FailSafeExpiryLengthSeconds field value is within a duration range of
+          0 to 65535"
```

Comment:
The YAML test to just read the attribute would automatically verify this, no?  Just because it would try to parse the result as the relevant payload and that parse would fail if that value were out of that range...

If that's all we are trying to verify, we don't need a manual test here.

Reply:
Makes sense , @manjunath-grl  can you just add this text as a pop-up for this step detailing why this particular step isn't in the Yaml and explain that as long as the payload is able to be parsed successfully it implicitly means the attribute verifies the range as well. Keep the text as a pop-up so users can know why we are doing this. Thanks !cc @kvikrambhat